simple_history/manager.py

```diff
@@ -51,10 +51,11 @@ def most_recent(self):
         excluded_fields = self.get_excluded_fields()
 
         for field in self.instance._meta.fields:
-            if isinstance(field, models.ForeignKey):
-                tmp.append(field.name + "_id")
-            elif field.name not in excluded_fields:
-                tmp.append(field.name)
+            if field.name not in excluded_fields:
```

Comment:
I might just change this so you don't nest the `if` statement.

Try:
```
if field.name in excluded_fields:
    continue
elif isinstance(field, models.ForeignKey):
    tmp.append(field.name + "_id")
else:
    tmp.append(field.name)
```

Reply:
Have you made the changes? If not maybe I can just do them and push it for you and it stays there?

Reply:
Yea can you do that? Then I’ll merge